aicsimageio/readers/ome_tiff_reader.py

```diff
@@ -118,7 +118,7 @@ def scenes(self) -> Tuple[str]:
     def _get_dims_and_coords_from_ome(
         ome: TiffTag,
         scene_index: int,
-    ) -> Tuple[List[str], Dict[str, Union[List, types.ArrayLike]]]:
```

Comment:
what happened to ArrayLike?

Reply:
Good catch. I swapped it for np.ndarray because `mypy` was complaining in general but should be fine to add back.